pkg/ivd/ivd_protected_entity_type_manager_test.go

```diff
@@ -86,10 +92,61 @@ func getVcConfigFromParams(params map[string]interface{}) (*url.URL, bool, error
 	return &vcUrl, insecure, nil
 }
 
+// TODO: remove. Used only in test.
```

Comment:
Was this something you meant to remove before checking in?  If not, let's change the comment to say something like "Used only for test, remove when tests are fixed" or something.